src/tasks/WorkloadBuildTasks/InstallWorkloadFromArtifacts.cs

```diff
@@ -221,6 +225,7 @@ private bool InstallPacks(InstallWorkloadRequest req, string nugetConfigContents
                                                         ["NUGET_PACKAGES"] = _nugetCachePath
                                                     },
                                                     logStdErrAsMessage: req.IgnoreErrors,
+                                                    silent: false,
```

Comment:
this looks intentional but I'll ask whether it is anyway :)

Reply:
Yep, it is. It will show the messages for the workload install step in the log. The earlier behavior was to emit the messages if there was a failure. Because the messages are emitted in `low` importance, so they won't show up for the user in quiet/normal builds.